taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -314,12 +352,20 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
                     )
                 } )
             }
-            {/* Dialog box for adding translations */}
+            {/* Button to add language to the list of shown languages or show the language if it already exists */}
+            <Button sx={{mt: 2}} onClick={handleOpen}>
+                Add/Show More Languages 
+                ({
+                    addedLanguageCodes.length - 
+                    addedLanguageCodes.filter(element => new Set(shownTranslationLanguages).has(element)).length
```

Comment:
nit: this `Set` can be `shownTranslationLanguages.includes(element)` . Also, if possible, let's rename `element` with something a bit more descriptive (`languageCodeItem`  or something)